tools/cli/commands/connect.py

```diff
@@ -293,7 +293,7 @@ def maybe_start(args, gcloud_compute, instance):
     return
 
 
-def run(args, gcloud_compute):
+def run(args, gcloud_compute, **unused_kwargs):
```

Comment:
Keyword args are being passed in. There has to be a corresponding **_something_ parameter, or that call will fail.